app/scripts/modules/kubernetes/src/container/resources.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="form-group">
+  <div class="col-md-3">
+  </div>
+  <div class="col-md-4 sm-label-left">
```

Comment:
I think this should be a `col-md-7`, and you should be able to remove the `div` above. 

The other thing to try would be `<div class="col-md-4 col-md-offset-3 sm-label-left">...</div>` (and still remove the `div` above).